### ISSUE TITLE
Add a parameter on the settings to be able to configure the url used by msm to get the list of the default skills

### ISSUE THREAD

**Issue by edaubert** [MycroftAI/mycroft-core#1380]
With version 0.9.14 of Mycroft, the url which define the default skills is hardcoded [fef914f](https://github.com/MycroftAI/mycroft-core/commit/fef914f771f5a9c307a3dcbe19fc0e4d4d8234d2).
Is it possible to replace it by a settings ?

Basically I have some skill I want to be removed (one of them is conflicting with another one I use).

When I remove the skill, it is always part of mycroft so I restart mycroft but as the default skills are hardcoded with the url, the removed skills which are part of the default list are reinstalled.

## First solution I see
We already do stuff like that on msm so I guess we can do the same to get the url to use for default skills definition.
```
mycroft_default_skill_url=$( jq -r '.skills.default_skills_url' /etc/mycroft/mycroft.conf )
```

However there are multiple locations for the configuration:
- home.mycroft.ai
- /etc/mycroft/mycroft.conf
- ~/.mycroft/mycroft.conf

Maybe we need to test every location to get the exact configuration.
However home.mycroft.ai seems to need authentication.

## Second solution I see
Maybe it would be better to use mycroft to send the url to use (and by default when there is no url use the hardcoded one).

In that case mycroft read the settings and so is able to get the url.

This could be done here: https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/skills/main.py#L258 by passing an url to the `default` command.

## Current workaround
Currently my workaround is to blacklist the skills I don't want.

**Comment by JarbasAl** [comment]
+1 for this

i have quite some skills meant to replace core ones (auto translatable stuff with no hope of becoming official, so no PRs for main skill repo)

**Comment by forslund** [comment]
This has been implemented and is now configurable as of #1577. Closing this issue.
